pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>11.4-SNAPSHOT</version>
+    <version>11.3</version>
```

Comment:
Not sure if we want this kind of change now. 🤔 
This is something to be done when performing a release or so. For the preview, we need also to align on a released package (even if it is only on preprod) but for development purposes (and specifically at this point in time) I think we still need to align on _snapshot_ version.

WDYT?

Reply:
PS: Guess it was related to https://jira.nuxeo.com/browse/NXP-29730 ? 😬

Reply:
indeed